recipe/meta.yaml

```diff
@@ -25,12 +24,10 @@ requirements:
   host:
     - python
     - scikit-build
-    - enum34  # [py2k]
```

Comment:
You need some special things to cross compile. It is likely easier to add this feedstock to the osx migrator so that it can do that for you automatically in a subsequent PR

Reply:
You can see some of the necessary changes here: but the bot may find others for you https://github.com/conda-forge/scikit-image-feedstock/blob/1567ed20b13f9b7e3e49de0c1c75059166035344/recipe/meta.yaml#L28

Reply:
Ok, thanks a lot! I removed this change for now, I'll try it again later

Reply:
You should just add the packages yo uwant to https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/migrations/osx_arm64.txt